olp-cpp-sdk-authentication/src/AuthenticationClient.cpp

```diff
@@ -19,1124 +19,23 @@
 
 #include "olp/authentication/AuthenticationClient.h"
 
-#include <rapidjson/document.h>
-#include <rapidjson/istreamwrapper.h>
-#include <rapidjson/stringbuffer.h>
-#include <rapidjson/writer.h>
-
 #include <chrono>
 #include <iomanip>
 #include <sstream>
```

Comment:
```suggestion
#include <utility>
```
Looks like we don't need these includes. Only std::move and SDK classes used here.
Ignore this if I am wrong ;-)

Reply:
Good catch. Resolved.